rkt/api_service.go

```diff
@@ -305,76 +343,194 @@ func getNetworks(p *pod) []*v1alpha.Network {
 	return networks
 }
 
-// getBasicPod returns *v1alpha.Pod with basic pod information.
-func getBasicPod(p *pod) *v1alpha.Pod {
-	pod := &v1alpha.Pod{Id: p.uuid.String(), Pid: -1}
+type errs struct {
+	errs []error
+}
+
+// Error is part of the error interface.
+func (e errs) Error() string {
+	return fmt.Sprintf("%v", e.errs)
+}
+
+func fillStaticAppInfo(st *imagestore.Store, p *pod, v1pod *v1alpha.Pod) error {
```

Comment:
Can you please add a docstring specifying that this will mutate `v1pod` in place, using appinfo from `p`?